packages/react-jsx-highcharts/src/components/BaseChart/BaseChart.js

```diff
@@ -1,55 +1,32 @@
 import React, { useState, useEffect, useRef, useLayoutEffect } from 'react';
 import PropTypes from 'prop-types';
-import debounce from '../../utils/debounce-raf';
-import { attempt } from 'lodash-es';
 import ChartContext from '../ChartContext';
-import { validChartTypes } from '../../utils/propTypeValidators';
 import usePrevious from '../UsePrevious';
+import { validChartTypes } from '../../utils/propTypeValidators';
+import createProvidedChart from './createProvidedChart';
+
+const noop = c => c;
 
 const BaseChart = ({
   children = null,
-  callback,
+  callback = noop,
   className = '',
   ...restProps
 }) => {
   const [rendered, setRendered] = useState(false);
   const domNodeRef = useRef(null);
   const chartRef = useRef(null);
-  const providerValueRef = useRef(null);
+  const providedChartRef = useRef(null);
 
   useLayoutEffect(() => {
     const myChart = initHighcharts(restProps, domNodeRef.current);
     chartRef.current = myChart;
+    providedChartRef.current = createProvidedChart(
```

Comment:
Move into `createProvidedChart` for consistency

Reply:
That's better.